compose/config/service_schema_v1.json

```diff
@@ -34,7 +34,12 @@
         "dns_search": {"$ref": "#/definitions/string_or_list"},
         "dockerfile": {"type": "string"},
         "domainname": {"type": "string"},
-        "entrypoint": {"$ref": "#/definitions/string_or_list"},
+        "entrypoint": {
+          "oneOf": [
+            {"type": "string"},
+            {"type": "array", "items": {"type": "string"}}
+          ]
+        },
```

Comment:
We're going to need this change in `compose/config/service_schema_v2.json` as well now, sorry about that.

Coping it over should be the only change.